src/components/WorkOrders/CloseWorkOrderFormComponents/FollowOnRequestTypeOfWorkForm.js

```diff
@@ -3,14 +3,26 @@ import { FOLLOW_ON_REQUEST_AVAILABLE_TRADES } from '../../../utils/statusCodes'
 import { Checkbox, TextArea } from '../../Form'
 import ErrorMessage from '../../Errors/ErrorMessage'
 import FollowOnRequestDifferentTradesForm from './FollowOnRequestDifferentTradesForm'
+import { useEffect, useState } from 'react'
 
 const FollowOnRequestTypeOfWorkForm = (props) => {
-  const { errors, register, getValues, setError, clearErrors, watch } = props
+  const {
+    errors,
+    register,
+    getValues,
+    setError,
+    clearErrors,
+    watch,
+    followOnData,
+  } = props
 
   const selectedFurtherWorkRequired =
     watch('followOnStatus') === 'furtherWorkRequired'
 
   const validateAtLeastOneOperativeOptionSelected = () => {
+
+    console.log("checkibn validation")
```

Comment:
console log :D

Reply:
I swear I removed all of these. maybe I didnt push chagnes

Reply:
mb my browser had the old version cached or smth on this one thinking: